app/src/main/java/com/example/android/weatherapp/WeatherAPIService.kt

```diff
@@ -18,8 +19,8 @@ private val retrofit =
 
 interface WeatherApiService {
 
-   @GET("data/2.5/weather?q=London,uk&appid=4dfdfc7144138f43bfa36b5b3a4b097f")
-   suspend fun getCurrentWeather(): Deferred<CurrentWeatherModel>
+   @GET("data/2.5/weather")
```

Comment:
Keep such APIs in a separate constants file, because more often than not, we are using these on multiple occassions.

Reply:
Create constant file in pull request#4